packages/roosterjs-content-model-plugins/lib/shortcut/shortcuts.ts

```diff
@@ -193,3 +197,46 @@ export const ShortcutDecreaseFont: ShortcutCommand = {
     },
     onClick: editor => changeFontSize(editor, 'decrease'),
 };
+
+/**
+ * Shortcut command for Intent list
+ */
+export const ShortcutIndentList: ShortcutCommand = {
+    shortcutKey: {
+        modifierKey: 'alt',
+        shiftKey: true,
+        which: Keys.ArrowRight,
+    },
+    onClick: editor => {
+        setShortcutIndentationCommand(editor, 'indent');
+    },
+};
+
+/**
+ * Shortcut command for Intent list
+ */
+export const ShortcutOutdentList: ShortcutCommand = {
+    shortcutKey: {
+        modifierKey: 'alt',
+        shiftKey: true,
+        which: Keys.ArrowLeft,
+    },
+    onClick: editor => {
+        setShortcutIndentationCommand(editor, 'outdent');
+    },
+};
+
+function setShortcutIndentationCommand(editor: IEditor, operation: 'indent' | 'outdent') {
```

Comment:
Let's put this function to a separate file.

Here we only declare shortcut commands.